.travis.yml

```diff
@@ -90,18 +90,26 @@ jobs:
       if: branch in (master, develop)
       python: 3.6
       install:
-        - pip install -r requirements/deploy.txt
-        - pip install --editable . # install wam, as needed by GB
+        - sudo docker pull ${DOCKER_USER}/${DOCKER_REPO}
```

Comment:
What do you think of just DOCKER_IMAGE or DOCKER_SLUG, rather than two separate variables?